catalog/config-controller-monitoring/prometheus-instance.yaml

```diff
@@ -62,21 +62,6 @@ spec:
     requests:
       # this may have to be increased depending on the number of metrics stored in the cluster.
       memory: 400Mi
-  # inject the stackdriver-prometheus sidecar, to pull the desired metrics.
-  containers:
-  - args:
-      - "--stackdriver.project-id=project-id"  # kpt-set: --stackdriver.project-id=${project-id}
```

Comment:
question: will the metrics still show up in regular cloud monitoring without this?

i.e. is this going to switch to GMP-only, or GMP + cloud monitoring availability?

Reply:
It will be both, the metrics will be available in cloud monitoring under `prometheus/metric-name`